src/main/resources/application-dev.yml

```diff
@@ -0,0 +1,13 @@
+spring:
+  profiles:
+    include: stdout
+
+  security:
+    oauth2:
+      resourceserver:
+        jwt:
+          public-key-location: classpath:local-public-key.pub
```

Comment:
This was previously used in other projects by ClientTrackingTelemetryModule to parse the JWT - but I think you are going with a different method that doesn't need the public key, so you can get rid of it.

Reply:
I think this needs to be done for ease of developing locally.

Reply:
I don't think you're using it anywhere.

Reply:
Wouldn't it be referenced implicitly by spring. e.g when it tries different methods for getting the public key?

Reply:
No I don't think so.  Spring has one of those @Conditionals to prevent that.  @petergphillips has taken it out of a couple of projects because it's not needed anymore.